modules/util/portal-tools-rest-builder-test-impl/src/main/java/com/liferay/portal/tools/rest/builder/test/internal/resource/v1_0/ERCAssetLibraryTestEntityResourceImpl.java

```diff
@@ -20,4 +22,26 @@
 )
 public class ERCAssetLibraryTestEntityResourceImpl
 	extends BaseERCAssetLibraryTestEntityResourceImpl {
+
+	@Override
+	protected Page<ERCAssetLibraryTestEntity>
+			doGetAssetLibraryERCAssetLibraryTestEntitiesPage(
+				String assetLibraryExternalReferenceCode)
+		throws Exception {
+
+		throw new UnsupportedOperationException(
```

Comment:
This is calling my attention, as you are implementing manually this with an exception saying this method needs to be implemented. I think maybe better to throw the exception without message